packages/terra-worklist-data-grid/src/WorklistDataGrid.jsx

```diff
@@ -62,138 +95,208 @@ function WorklistDataGrid(props) {
     ariaLabel,
     columns,
     rows,
+    onCellSelect,
+    onRowSelect,
+    onClearSelectedRows,
+    onClearRowSelectionMode,
+    hasSelectableRows,
+    intl,
   } = props;
 
+  const grid = useRef();
   const focusedRow = useRef(0);
   const focusedCol = useRef(0);
-  const grid = useRef();
+  const rowSelectionMode = useRef(false); // used to detect change in row selection Mode.
```

Comment:
If this variable is detecting change in row selection mode, should it be a state variable?  Will changing this value toggle the column in the table?